atomic_reactor/constants.py

```diff
@@ -60,3 +60,14 @@
 )
 
 DEFAULT_DOWNLOAD_BLOCK_SIZE = 10 * 1024 * 1024 # 10Mb
+
+LABEL_ALIASES = {
+    "name" : ('name', 'Name'),
```

Comment:
I'd rather these were symbolic names in some way, to make it clear while reading code that this is not the real label.